scripts/src/main/resources/scripts/devon-autocomplete

```diff
@@ -0,0 +1,34 @@
+#/usr/bin/env bash
+_devon_completions()
+{
+  local cur commandlist options
+  COMPREPLY=()
+  source ~/.devon/home
```

Comment:
here we also have the case as above but this time it is not easy to solve.
Can you try to research in which context this `_devon_completions` function is invoked? Is it possible to determine the current IDE installation based on `$PWD` like it is done in `environment-project` script?

Reply:
It works very well with the same command `$(dirname "${BASH_SOURCE:-$0}")`. It returns me the path where the autocompletion script is located. (see next commit)

Reply:
The autocomplete script is registered globally on the first setup of your devonfw-ide. The following things can happen:
* A newer version of devonfw-ide is installed in a separate folder that has additional commands. Those will then not be available for autocompletion
* After additional installation(s) of `devonfw-ide` have been added the first initial devonfw-ide installation was removed. The config written to `.bashrc` or `.zshrc` was pointing to that installation with an absolute path and therefore will be broken.

So to make it "perfect" we would need to do the same trick as we did with `devon[.bat]` script:
* create a minimal but stable and universal script as shim also for `devon-autocomplete`
* this can be installed to a central installation (`~/scripts` just like with `devon.bat` or `~/.devon`) on initial setup and the be registered to `.bashrc` and `.zshrc`.
* This `devon-autocomplete` script itself would work portable such that is uses the current `PWD` where the user is currently located to find `DEVON_IDE_HOME` (see `environment-project`).